packages/mwp-app-route-plugin/src/index.js

```diff
@@ -35,11 +34,11 @@ export default function register(
 		);
 	});
 	// set up launchdarkly instance before continuing
-	server.on('stop', ldClient.close);
-	ldClient.once(`ready`, () => next());
```

Comment:
how can we force the server to pause until the `ldClient` is ready? can you return a promise from the `register` function?